clients/filesystem-hadoop3/src/test/java/com/datastrato/gravitino/filesystem/hadoop3/TestSimpleClient.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.filesystem.hadoop3;
+
+import org.junit.jupiter.api.BeforeAll;
+
+public class TestSimpleClient extends TestGvfsBase {
+  @BeforeAll
+  public static void setup() {
+    TestGvfsBase.setup();
+    conf.set(
+        GravitinoVirtualFileSystemConfiguration.FS_GRAVITINO_CLIENT_AUTH_TYPE_KEY,
+        GravitinoVirtualFileSystemConfiguration.SIMPLE_AUTH_TYPE);
+  }
```

Comment:
Could you verify the token from the client satisfies the expect value?

Reply:
Do you mean check the simple auth's token?

Reply:
> Do you mean check the simple auth's token?

Yes.

Reply:
Done